src/ConfigGen/Program.cs

```diff
@@ -621,6 +625,13 @@ static List<PropertySpecification> choosePreferredNames(List<PropertySpecificati
             }).ToList();
         }
 
+        static void PrintProps(IEnumerable<PropertySpecification> props)

```

Comment:
these changes make debugging easier. since this code isn't very robust to changes in librdkafka config docs, I expect more debugging will be needed in the future, so leaving these changes in.